### ISSUE TITLE
verifyEditable and verifyNotEditable on a button not working

### ISSUE THREAD

**Issue by sentry0** [vmi/selenese-runner-java#227]
Both of these commands work as expected when running via Selenium IDE.  

Take the same script and run it via selenese-runner and they fail.
```

<tr>
	<td>verifyEditable</td>
	<td>id=my-sweet-button-id</td>
	<td></td>
</tr>
```

**Comment by vmi** [comment]
I fixed this problem and released 3.2.0.
Please test it.

**Comment by vmi** [comment]
Hi,
If your problem is not resolved in the current release, please reopen this issue.
